frontend_v2/src/styles/base.scss

```diff
@@ -346,6 +346,7 @@ mat-chip {
 
 .update-page {
   display: block;
+  z-index: 1;
```

Comment:
Please don't make this change in the base class, instead you should override the same class under the leaderboard component.

Reply:
We are using this class only for the refresh label and we are using this class in two components where we would be facing this problem, that's why I did the changes in `base` class.

Reply:
Okay, so if we require this change at both places, it's totally fine to change it in here.